Automattic/jetpack
# E2E tests: Add e2e jetpack connect flow

To be able to use/test Gutenpack blocks - we need to have a Jetpack connected site with a paid plan. This PR adds a connection & purchase flow to solve this configuration process.   

#### Changes proposed in this Pull Request:

* Adds a bunch of stuff!
* Adds a whole Jetpack connection flow with plan purchase

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* p9dueE-Ju-p2

#### Testing instructions:






* TBD
* Check travis E2E job
* decrypt config. Check the README file for the instructions. Are they hard to follow? 
* Make sure you have a running docker WP install with an activated jetpack (but not connected).
* run `yarn test-e2e` locally. use `PUPPETEER_HEADLESS=false` env variable to see the tests in action. Maybe check docs how to set up the tests

#### Proposed changelog entry for your changes:

* E2E: add Jetpack connect flow
